syntax/c.vim

```diff
@@ -1,7 +1,7 @@
 " Vim syntax file
 " Language:	C
-" Maintainer:	Bram Moolenaar <Bram@vim.org>
-" Last Change:	2016 Nov 18
+" Current Maintainer:	vim-jp (https://github.com/vim-jp/vim-cpp)
```

Comment:
I guess I'm trying to indicate that vim-jp is the source of modifications to c.vim (to explain differences from Bram's version). 

Would something else be better:

> Patches From:	vim-jp (https://github.com/vim-jp/vim-cpp)